plugins/MauticFocusBundle/Tests/Model/FocusModelFunctionalTest.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MauticPlugin\MauticFocusBundle\Tests\Model;
+
+use Mautic\CoreBundle\Test\MauticMysqlTestCase;
+use Mautic\LeadBundle\Entity\Lead;
+use Mautic\PageBundle\Entity\Hit;
+use MauticPlugin\MauticFocusBundle\Entity\Focus;
+use MauticPlugin\MauticFocusBundle\Entity\Stat;
+use MauticPlugin\MauticFocusBundle\Model\FocusModel;
+
+class FocusModelFunctionalTest extends MauticMysqlTestCase
+{
+    public function testViewsCount(): void
+    {
+        /** @var FocusModel $focusModel */
+        $focusModel = self::$container->get('mautic.focus.model.focus');
```

Comment:
As commented above, I'd change these tests to calling the new controller actions to test the whole feature like a user would instead of calling just the model method.